BEP128.md

```diff
@@ -0,0 +1,48 @@
+# BEP-128: Improve Binance Smart Chain Staking Reward Distribution
+
+- [BEP-128: Improve Binance Smart Chain Staking Reward Distribution](#bep-128-improve-binance-smart-chain-staking-reward-distribution)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+  - [4. Motivation](#4-motivation)
+  - [5. Specification](#5-specification)
+    - [5.1 Overall Workflow](#51-overall-workflow)
+    - [5.2 Distribution Batch Size](#52-distribution-batch-size)
+    - [5.3 User Impact](#53-user-impact)
+  - [6. License](#6-license)
+
+## 1. Summary
+This BEP describes a proposal to improve the Binance Smart Chain staking reward distribution mechanism on Binance Chain.
+
+## 2. Abstract
+Instead of distributing BSC staking rewards in a single block for each day, this proposal suggests distributing staking rewards in many consecutive blocks, to avoid heavy burden on any specific blocks.
+
+## 3. Status
+This BEP is a draft.
+
+## 4. Motivation
+Currently, BSC’s staking rewards are distributed in the first block of a day on Binance Chain (which is named breath block). With the increasing number of delegators (more than 50,000 nowadays), it will lead to a heavy load to breath block, and users' transactions/requests could be affected. Meanwhile, it could also be a bottleneck for further evaluation of Binance Chain. Thus, this proposal provides a solution to fix the issue and benefit the evaluation of Binance Chain in the long term.
+
+## 5. Specification
+### 5.1 Overall Workflow
+![overall workflow](https://user-images.githubusercontent.com/61674316/148506577-aac9d05a-488d-4ff0-b879-1aa1a0e5364c.png)
```

Comment:
can we create an asset folder to place the images instead of referring the images outside

Reply:
like this https://github.com/ethereum/EIPs/tree/master/assets

Reply:
sure